src/utils/Users.js

```diff
@@ -0,0 +1,133 @@
+const niceJoin = array =>
```

Comment:
It took me a bit to understand what `niceJoin` does. Maybe we could rename the function to something like `formatListToLanguage`.